atlassian-terraform/atlassian.auto.tfvars

```diff
@@ -0,0 +1,62 @@
+# Sumo Logic - Atlassian Terraform
+# Configure Jira Cloud, Jira Server, OpsGenie, BitBucket credentials and parameters.
+
+
+# Jira Cloud
+jira_cloud_url      = "https://arunpatyal.atlassian.net"
+jira_cloud_user     = "arunpatyal0809@gmail.com"
+jira_cloud_password = "AVQYnQwD1IdI2GylyUQx99B6" # Password or API Key, To generate API Key: https://confluence.atlassian.com/cloud/api-tokens-938839638.html
+jira_cloud_jql      = ""                         #jql = "project = Sumo" See, https://support.atlassian.com/jira-software-cloud/docs/what-is-advanced-searching-in-jira-cloud/
+// See https://developer.atlassian.com/cloud/jira/platform/webhooks/ for supported events
+jira_cloud_events = ["jira:issue_created", "jira:issue_updated", "jira:issue_deleted",
+  "issue_property_set", "issue_property_deleted", "comment_created", "comment_updated", "comment_deleted",
+  "worklog_created", "worklog_updated", "worklog_deleted",
+  "attachment_created", "attachment_deleted", "issuelink_created", "issuelink_deleted",
+  "project_created", "project_updated", "project_deleted",
+  "jira:version_released", "jira:version_unreleased", "jira:version_created", "jira:version_moved", "jira:version_updated", "jira:version_deleted", "jira:version_deleted",
+  "user_created", "user_updated", "user_deleted",
+  "option_voting_changed", "option_watching_changed", "option_watching_changed", "option_unassigned_issues_changed", "option_subtasks_changed", "option_attachments_changed", "option_issuelinks_changed", "option_timetracking_changed",
+  "sprint_created", "sprint_deleted", "sprint_updated", "sprint_started", "sprint_closed",
+  "board_created", "board_updated", "board_deleted", "board_configuration_changed",
+"jira_expression_evaluation_failed"]
+
+
+# Jira Server
+# This script configures Jira Server WebHooks. Jira Server Logs collection needs to be configured as explained in Step 1 [here](https://help.sumologic.com/07Sumo-Logic-Apps/08App_Development/Jira/Collect_Logs_for_Jira#step-1-set-up-local-file-sources-on-an-installed-collector).
+# Configure the log collection and update the variable jira_server_access_log_category with the selected source category.
+jira_server_access_logs_sourcecategory = "Atlassian/Jira/Server*"
+jira_server_url                        = "http://localhost:8080"
+jira_server_user                       = "arunpatyal0809"
+jira_server_password                   = "Qazwsx@2!" # Needs to be the password. API Key is not supported on Jira Server yet: https://jira.atlassian.com/browse/JRASERVER-67869?_ga=2.198461357.302520551.1583314185-1454539139.1580206139&error=login_required&error_description=Login+required&state=d3142ec3-6eb1-4207-bdd7-ce6b93900aa1
```

Comment:
ANother password leak, but not as important unless you reuse.

Reply:
Wrong file was uploaded. Correct file with placeholders has been uploaded and all the credentials are revoked.